core/domain/auth_jobs_one_off.py

```diff
@@ -209,13 +208,9 @@ def report_assocs_missing(cls, assoc_info_pairs):
     def report_assoc_collisions(cls, auth_id, assoc_info_pairs):
         """Yields debug information for associations mapped to the same auth_id.
 
-        NOTE: Since assoc_by_auth_id and Firebase accounts are keyed by auth_id,
-        and are therefore guaranteed to have no collisions, the associations
-        reported will always come from UserAuthDetailsModel.
-
         Args:
             auth_id: str. The auth_id to check.
-            assoc_info_pairs: list(tuple(str, (str, bool))). The list of
+            assoc_info_pairs: list(tuple(str, (str, bool)|bool)). The list of
```

Comment:
Why could it also be bool? Better to keep it single-type, if possible...

Reply:
Done.